README.md

```diff
@@ -1 +1,420 @@
-# identity-agent-sso
\ No newline at end of file
+# WSO2 SAML SDK for Java
+
+The WSO2 Security Assertions Markup Language (SAML) Software Development Kit (SDK) for Java enables software
+ developers to integrate SAML based SSO authentication with Java Web applications. The SDK is built on top of the
+  OpenSAML library which allows Java developers to develop cross-domain single sign-on and federated access control
+   solutions with minimum hassle.
+
+## Getting Started
+
+These instructions will guide you on integrating SAML into your Java application with the WSO2 SAML SDK.
+This allows the developers to turn a Java application into a SP (Service Provider) that can be connected to an IdP
+ (Identity Provider) which can support the following main features among many others.
+
+- Single Sign-On (SSO) and Single Log-Out (SLO) (SP-Initiated and IdP-Initiated).
+- Assertion and nameID encryption.
+- Assertion signatures.
+- Message signatures: AuthNRequest, LogoutRequest, LogoutResponses.
+- Enable an Assertion Consumer Service endpoint.
+- Enable a Single Logout Service endpoint.
+- Publish the SP metadata.
+
+### Trying out the sample
+A sample app for demonstrating SAML based SSO authentication, SLO and attribute retrieval is hosted at:
+https://github.com/wso2-extensions/identity-agent-sso/tree/master/resources/SampleApp
+
+You can either download the pre-built SampleApp.war from here[link] or build it from the source:
+
+#### Building from the source
```

Comment:
Let's have separate section on building the whole repo in a later section. Building specifically the sample from the source is not needed

Reply:
+1. Thanks. The comments have been addressed.